src/main/java/io/confluent/idesidecar/websocket/messages/ConnectionEventBody.java

```diff
@@ -0,0 +1,31 @@
+package io.confluent.idesidecar.websocket.messages;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.confluent.idesidecar.restapi.connections.ConnectionState;
+import io.confluent.idesidecar.restapi.models.ConnectionSpec.ConnectionType;
+import io.confluent.idesidecar.restapi.models.ConnectionStatus;
+
+/**
+ * Sent by the sidecar to the workspace when a connection is created, when existing connections
+ * are updated or their status changed, or when a connection is deleted.
+ */
+public record ConnectionEventBody(
+    @JsonProperty("connection_id") String connectionId,
+    @JsonProperty("connection_type") ConnectionType connectionType,
+    @JsonProperty("event_kind") EventKind eventKind,
+    @JsonProperty("connection_status")ConnectionStatus connectionStatus
```

Comment:
What about adding the entire [Connection](https://github.com/confluentinc/ide-sidecar/blob/main/src/main/java/io/confluent/idesidecar/restapi/models/Connection.java) model, including the spec, to the event instead of sending the `connectionId`, `connectionType`, and `connectionStatus` as separate fields?

Also, do we want to add a timestamp field that holds the time when the event has been created?

Reply:
I didn't add a timestamp or sequence number. Let's add those if/when needed.